cmd/swapcli/main.go

```diff
@@ -308,8 +312,8 @@ func cliApp() *cli.App {
 				},
 			},
 			{
-				Name:    "suggested-exchange-rate",
-				Aliases: []string{"price-feed"},
+				Name:    "price-feed",
+				Aliases: []string{"suggested-exchange-rate"},
```

Comment:
I'm open to other names. We do report both prices, not just the suggested exchange rate. `suggested-exchange-rate` was too much typing for me. It is kind of long to even keep around as an alias.

This is what we print:
```text
Exchange rate: 0.08537
XMR/USD Price: 162.21        (2023-06-25 18:12:03 -0500 CDT)
ETH/USD Price: 1900.08429046 (2023-06-25 18:20:39 -0500 CDT)
```
One positive thing about the switch to the Chainlink feed on Optimism for XMR is that it is updated at a much higher frequency. I think the old (now dead) feed was only getting updated like every 4 hours, while the new one appears to be every 15 minutes.